Proiect/src/Cinema/Adresa.java

```diff
@@ -0,0 +1,58 @@
+package Cinema;
```

Comment:
[Fix] toate clasele in java trebuie sa vina sub src, apoi sub un pachet care reprezinta reversul domeniului organizatiei careia ii apartine aplicatia, ex: ro.unibuc, co.uk.bbc, com.amazon. Noi ca exemple am folosit com.company